swap/src/protocol/alice/swap.rs

```diff
@@ -39,15 +50,23 @@ pub async fn run_until(mut swap: Swap, exit_early: fn(&AliceState) -> bool) -> R
     Ok(current_state)
 }
 
-async fn next_state(
+async fn next_state<LR>(
     swap_id: Uuid,
     state: AliceState,
     event_loop_handle: &mut EventLoopHandle,
     bitcoin_wallet: &bitcoin::Wallet,
     monero_wallet: &monero::Wallet,
     env_config: &Config,
-) -> Result<AliceState> {
-    info!("Current state: {}", state);
+    mut rate_service: LR,
+) -> Result<AliceState>
+where
+    LR: LatestRate,
+{
+    let rate = rate_service
+        .latest_rate()
+        .map_or("NaN".to_string(), |rate| format!("{}", rate));
+
+    info!(%state, %rate, "Update");
```

Comment:
Instead of "Update", maybe "Advancing state" is more descriptive?

Reply:
@bonomat If you resolve conversations without changing the code, can you please add a comment why?

Reply:
I changed it to what you proposed: https://github.com/comit-network/xmr-btc-swap/blob/088ecc47923f04bd348764b7b341ae4261c49739/swap/src/protocol/alice/swap.rs#L69

Reply:
Huh okay, sorry about that. Somehow that didn't show up in my diff.